[#58113] 14.6 Documentation: Progress reporting changes  https://comm… (#16869)
opf/openproject: docs/system-admin-guide/manage-work-packages/work-package-progress-tracking/README.md

---

sidebar_navigation:
  title: Progress tracking
  priority: 965
description: Manage Work package progress tracking.
keywords: work package progress tracking, percentage complete, % complete
---

# Manage work package progress tracking

> To manage the settings for progress tracking in work packages, navigate to  *Administration* → *Work packages → Progress tracking*. 

Reviewer: ```suggestion
To manage the settings for progress tracking in work packages, navigate to  *Administration* → *Work packages* → *Progress tracking*. 
```

![Progress tracking settings under OpenProject administration](openproject_system_guide_progress_tracking_settings.png)

## Progress calculation mode
*Progress calculation mode* lets you select between *work-based* and *status-based* modes.

> - **Work-based mode**: *%&nbsp;Complete* is either set manually or is automatically calculated based on *Work* and *Remaining work* values for that work package, both of which are then necessary to have a value for %&nbsp;Complete. Please refer to [progress tracking user guide](../../../user-guide/time-and-costs/progress-tracking/#work-based-progress-reporting) for more details and calculation examples.

Reviewer 2: Minor suggestion for change:

> - **Work-based mode**: *%&nbsp;Complete* is either set manually or calculated automatically based on the values for *Work* and *Remaining work*, if they exist. Please refer to [progress tracking user guide](../../../user-guide/time-and-costs/progress-tracking/#work-based-progress-reporting) for more details and calculation examples.

> - **Work-based mode**: *%&nbsp;Complete* is either set manually or is automatically calculated based on *Work* and *Remaining work* values for that work package, both of which are then necessary to have a value for %&nbsp;Complete. Please refer to [progress tracking user guide](../../../user-guide/time-and-costs/progress-tracking/#work-based-progress-reporting) for more details and calculation examples.

Author: ```suggestion
- **Work-based mode**: *%&nbsp;Complete* is either set manually or is automatically calculated based on *Work* and *Remaining work*, if they exist. Please refer to [progress tracking user guide](../../../user-guide/time-and-costs/progress-tracking/#work-based-progress-reporting) for more details and calculation examples.
```

- **Status-based mode**: you will have to define fixed %&nbsp;Complete values for each [work package status](../work-package-status), which will update automatically when team members update the status of their work packages.

 > [!NOTE]
 > When switching progress calculation mode from one to another, you will see a warning message.
 >
 > - Changing progress calculation mode from work-based to status-based will result in all existing *% Complete* values to be lost and replaced with values associated with each status. Existing values for *Remaining work* may also be recalculated to reflect this change. This action is not reversible.
 >

>  > - Changing progress calculation mode from status-based to work-based will make % Complete a non-editable field whose value is derived from Work and Remaining work. Existing values for % Complete are preserved. If values for Work and Remaining work were not present, they will be required in order to change % Complete.

Reviewer: The text is not up-to-date as it says that % complete is non-editable.

Here is a suggestion (copy-pasted from the updated warning text in the app)

```suggestion
 > - Changing progress calculation mode from status-based to work-based will make the *% Complete* field freely editable. If you optionally enter values for *Work* or *Remaining work*, they will also be linked to *% Complete*. Changing *Remaining work* can then update *% Complete*.
```
